src/uu/free/src/free.rs

```diff
@@ -122,7 +123,7 @@ fn parse_meminfo() -> Result<MemInfo, Box<dyn std::error::Error>> {
 
 // print total - used - free combo that is used for everything except memory for now
 // free can be negative if the memory is overcommitted so it has to be signed
-fn tuf_combo<F>(name: &str, total: u64, used: u64, free: i128, f: F)
+fn gen_tuf_combo_str<F>(name: &str, total: u64, used: u64, free: i128, f: F) -> String
```

Comment:
I think `construct` better than `gen`?

```suggestion
fn construct_tuf_combo_str<F>(name: &str, total: u64, used: u64, free: i128, f: F) -> String
```